user/languages/elixir.md

```diff
@@ -44,6 +44,30 @@ language: elixir
 ```
 {: data-file=".travis.yml"}
 
+### Specify which Elixir version to test with
```

Comment:
We should probably default to to 'build' instead of 'test' here?

Reply:
Agreed.